pkgs/applications/window-managers/sway/default.nix

```diff
@@ -1,40 +1,57 @@
-{ stdenv, fetchFromGitHub
-, cmake, pkgconfig, asciidoc, libxslt, docbook_xsl
-, wayland, wlc, libxkbcommon, pcre, json_c, dbus
-, pango, cairo, libinput, libcap, pam, gdk_pixbuf, libpthreadstubs
-, libXdmcp
+{ stdenv, fetchFromGitHub, fetchpatch
+, meson, ninja
+, pkgconfig, scdoc
+, wayland, libxkbcommon, pcre, json_c, dbus, libevdev
+, pango, cairo, libinput, libcap, pam, gdk_pixbuf
+, wlroots, wayland-protocols
 , buildDocs ? true
 }:
```

Comment:
Can we just build the docs unconditionally into a separate output?

Reply:
@jtojnar Turning this conditional off helped when I've developed my patch for 0.x branch of sway. Docs were building slow by some reason, postponing test times.
I don't see such issue now, suggest to drop it and always build docs.

Reply:
That's because sway uses scdoc now instead of asciidoc.

Reply:
I agree with @gnidorah it's probably best to simply drop `buildDocs` for now as it is not that relevant anymore.